playbooks/collect_debug.yml

```diff
@@ -3,7 +3,7 @@
   become: true
   vars:
     bats_output_dir: '/root/bats_results'
-    remote_dir: "/tmp/debug-{{ pipeline_type | default('foreman') }}-{{ pipeline_version | default('nightly') }}-{{ pipeline_os | default('el7') }}"
+    remote_dir: "/tmp/debug-{{ pipeline_type | default('foreman') }}-{{ pipeline_version | default('nightly') }}-{{ pipeline_os | default('el7') }}-{{ pipeline_action | default('install') }}"
```

Comment:
Thoughts about storing it in `/tmp/forklift-debug-$user/$pipeline`. Then you could also collect `/tmp/forklift-debug-*/**`, without knowing the exact pipeline.

Reply:
I did (not exactly `forklift-debug-$user` but similar) and opted for "let's not change too much right now"